tests/errors/terror_in_tuple_cons.test

```diff
@@ -0,0 +1,5 @@
+discard """
+  description: "Either the if or else branch has an error"
```

Comment:
```suggestion
  description: "Error in a field of a tuple"
```

I believe this is what you might have meant

Reply:
Oh, I forgot to update the description after duplicating the if test, thanks.